flixel/graphics/frames/FlxAtlasFrames.hx

```diff
@@ -421,3 +509,93 @@ typedef TexturePackerObject =
 {
 	frames:Dynamic
 }
+
+typedef AsepriteData =
+{
+	var frames:Dynamic,
```

Comment:
i think typdefs either need to be one of the following
```hx
{ foo:String, bar:String}
```
or
```hx
{ var foo:String; var bar:String; }
```
notice the ;

try to use your branch of flixel in a test project to make sure it compiles

Reply:
Ok